azurerm_public_ip  - Basic SKU in public ip does not support Availability Zone (#7017)
terraform-providers/terraform-provider-azurerm: website/docs/r/public_ip.html.markdown

* `zones` - (Optional) A collection containing the availability zone to allocate the Public IP in.

> -> **Please Note**: Availability Zones are [only supported in several regions at this time](https://docs.microsoft.com/en-us/azure/availability-zones/az-overview). And only [Standard SKU](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-ip-addresses-overview-arm#standard) public IP support Availability Zone scenarios.

Reviewer: This should be 
```suggestion
-> **Please Note**: Availability Zones are only supported with a [Standard SKU](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-ip-addresses-overview-arm#standard) and [in select regions](https://docs.microsoft.com/en-us/azure/availability-zones/az-overview) at this time.
```
